tests/test_server/test_apps/test_identity/test_models.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any
+
+import pytest
+
+from server.apps.identity.models import _UserManager  # noqa: WPS450
+
+
+class TestUserManager:  # noqa: WPS306
```

Comment:
Тут возникают проблемы если указать родительский класс `object`?

```suggestion
class TestUserManager(object):
```